src/fixtures/logstash_fields.js

```diff
@@ -51,6 +53,10 @@ function stubbedLogstashFields() {
       scripted = !!script,
     } = metadata;
 
+    const type = (esType === 'conflict' || esType === 'unknown')
```

Comment:
Correct, `unknown` is not possible here. The `conflict` type isn't from es, but is a special kbnFieldType, but we don't have any other way of representing here.